examples/dt/nova/nova-multiplecells/values.yaml

```diff
@@ -0,0 +1,130 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: edpm-nodeset-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  ssh_keys:
```

Comment:
I am not a fan of hacing ssh_keys set in two values files in this PR. The nodeset in the library will create the same secret `dataplane-ansible-ssh-private-key-secret` for both nodesets. If a user looks at this they may be led to believe they can use different keys for the (I assume pre-provisioned nodes?) when using this DT - but that will not work since applying the second nodeset will modify the same secret instead of creating a new one.

Reply:
I'm not familiar with how the common template interacts with dt/va variant templates.  If the ssh keys are not present in the DT/VA's will [common](https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/ci_gen_kustomize_values/templates/common/edpm-nodeset-values/values.yaml.j2#L14) add in the keys?

Reply:
Ack I can removed these.

Reply:
I am not sure you can remove them, if you cannot - then this change should not be blocked by this comment from me.

I was trying to point out an issue with the library not supporting multiple nodesets in a good way. Essentilly you have two nodesets referencing lib/dataplane/nodeset/kustomization.yaml#L6-L8 - so you end up applying two separate files defining the same secret resource.